src/assets/translations/en/main.json

```diff
@@ -477,6 +477,7 @@
     "earnBalance": "Earn Balance",
     "rewardsBalance": "Rewards Balance",
     "earnBody": "Earn passive income by staking your assets or depositing them into a DeFi strategy.",
+    "noAccountsOpportunities": "You have no accounts for this asset, so staking opportunities are currently unavailable.",
```

Comment:
@shapeshift/product to confirm on the copy/flow

Reply:
It's been validated by @twblack88 already 👍